src/Reflection/Php/EnumCasesMethodReflection.php

```diff
@@ -116,4 +116,9 @@ public function getSelfOutType(): ?Type
 		return null;
 	}
 
+	public function returnsByReference(): TrinaryLogic
+	{
+		return TrinaryLogic::createMaybe();
```

Comment:
This is a built-in PHP method, we should be able to tell if it's yes or no.

Reply:
the method cannot be overridden, so I hardcode NO
https://3v4l.org/ebm83